neuro_flow/parser.py

```diff
@@ -1353,6 +1353,39 @@ def parse_action_main(ctor: BaseConstructor, node: yaml.MappingNode) -> ast.Base
 ActionLoader.add_constructor("action:main", parse_action_main)  # type: ignore
 
 
+# #### Action parser ####
```

Comment:
The idea of this overengineering is to make it easier to implement more "smart" metafiles.